src/patternfly/components/FormControl/examples/FormControl.md

```diff
@@ -123,129 +121,63 @@ import './FormControl.css'
-Success
-{{/form-control}}
-<br><br>
-{{#> form-control controlType="textarea" form-control--modifier="pf-m-warning pf-m-icon-sprite" form-control--attribute='name="textarea-warning" id="textarea-warning-sprite" aria-label="Warning state textarea example"'}}
-Warning
-{{/form-control}}
-<br><br>
-{{#> form-control controlType="textarea" form-control--modifier="pf-m-icon-sprite" form-control--attribute='required name="textarea-error" id="textarea-error-sprite" aria-label="Error state textarea example" aria-invalid="true"'}}
-Error
-{{/form-control}}
-
-```
-
 ## Documentation
 
 ### Accessibility
 | Attribute | Applied to | Outcome |
 | -- | -- | -- |
-| `id` | `.pf-v5-c-form-control` | Provides an `id` value that can be used with the `for` attribute on an associated `<label>` element to provide an accessible label for the form control element.
-| `aria-invalid="true"` | `.pf-v5-c-form-control` | Indicates that the form control is in the error state and applies error state styling. |
-| `aria-label="descriptive text"` | `.pf-v5-c-form-control` | Provides an accessible label for assistive technology. |
-| `aria-expanded="true"` | `.pf-v5-c-form-control.pf-m-expanded` | Indicates that clicking in the form control has toggled something else to be expanded. |
+| `id` | `.pf-v5-c-form-control > input`,`.pf-v5-c-form-control > select`, or `.pf-v5-c-form-control > textarea` | Provides an `id` value that can be used with the `for` attribute on an associated `<label>` element to provide an accessible label for the form control element.
+| `aria-invalid="true"` | `.pf-v5-c-form-control > input`,`.pf-v5-c-form-control > select`, or `.pf-v5-c-form-control > textarea` | Indicates that the form control is in the error state and applies error state styling. Use with `.pf-m-error` on the `.pf-v5-c-form-control` to ensure correct styling. |
+| `aria-label="descriptive text"` | `.pf-v5-c-form-control > input`,`.pf-v5-c-form-control > select`, or `.pf-v5-c-form-control > textarea` | Provides an accessible label for assistive technology. |
+| `aria-expanded="true"` | `.pf-v5-c-form-control > input` | Indicates that clicking in the form control has toggled something else to be expanded. Use with `.pf-m-expanded` on the `.pf-v5-c-form-control` to ensure correct styling. |
 
 ### Usage
 | Class | Applied to | Outcome |
 | -- | -- | -- |
-| `.pf-v5-c-form-control` | `<input>`,`<textarea>`, `<select>` |  Initiates an input, textarea or select. For styling of checkboxes or radios see the [checkbox component](/components/checkbox) or [radio component](/components/radio). **Required**  |
+| `.pf-v5-c-form-control` | `<div>` |  Initiates a container for an input, textarea or select. For styling of checkboxes or radios see the [checkbox component](/components/checkbox) or [radio component](/components/radio). **Required**  |
+| `.pf-v5-c-form-control` | `<input>`,`<textarea>`, `<select>` |  Initiates an input, textarea or select.  **Required**  |
```

Comment:
Is the intent that either a div or input/textarea/select can have this class? Based on the examples it seems like line 172 here should be removed.